src/actor/ActorTemplate.js

```diff
@@ -330,17 +408,36 @@ exports = function(inherits) {
       });
     };
 
+    /** Shortcut to this.view.style.opacity
+        @var {number} ActorTemplate#opacity */
     this._addStyleProperty('opacity');
+    /** Shortcut to this.view.style.fliipX
+        @var {number} ActorTemplate#fliipX */
     this._addStyleProperty('flipX');
+    /** Shortcut to this.view.style.flipY
+        @var {number} ActorTemplate#flipY */
     this._addStyleProperty('flipY');
+    /** Shortcut to this.view.style.r
+        @var {number} ActorTemplate#rotation */
     this._addStyleProperty('r', 'rotation');
+    /** Shortcut to this.view.style.zIndex
+        @var {number} ActorTemplate#zIndex */
     this._addStyleProperty('zIndex');
 
-    Object.defineProperty(this, "currentAnimation", {
-      get: function() { return this.view.isSprite ? this.view._currentAnimationName : ""; }
+    /**
+     * The current sprite animation, or an empty string if this is not a sprite
+     * @var {number} ActorTemplate#currentAnimation
+     * @readonly
+     */
+    Object.defineProperty(this, 'currentAnimation', {
+      get: function() { return this.view.isSprite ? this.view._currentAnimationName : ''; }
     });
 
-    Object.defineProperty(this, "scale", {
+    /**
```

Comment:
@lunarraid Currently we call `applyScaledBounds`, which updates `this.model.hitBounds`.  Can you clarify "At the moment, this does not update bounds on set."?

@rogueSkib https://github.com/gameclosure/devkit-scene/issues/34